src/frontend/src/components/Header/index.tsx

```diff
@@ -1,36 +1,51 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React, {useState, useEffect} from "react";
+import { Link, useLocation } from "react-router-dom";
 import Logo from "../Logo";
+import { isLoggedIn, oeciLogout } from "../../service/cookie-service";
 
-export default class Header extends React.Component {
-  public render() {
-    return (
-      <div className="bg-white shadow">
-        <nav 
-          className="mw8 relative center flex flex-wrap justify-between pa3"
-          aria-label="Primary"
-        >
-          <div className="logo mb4 mb0-ns">
-            <Link to="/" aria-label="Home">
-              <Logo />
-            </Link>
-          </div>
-          <div className="mt5 mt2-ns">
-            <Link
-              to="/manual"
-              className="link hover-blue f5 fw6 pv2 ph0 ph3-ns mr4-ns"
-            >
-              Manual
-            </Link>
-            <Link
-              to="/record-search"
-              className="absolute top-1 right-1 static-ns bg-blue white bg-animate hover-bg-dark-blue f5 fw6 br2 pv2 ph3"
+
+export default function Header (){
+  const [loggedIn, setLoggedIn] = useState(isLoggedIn());
+  const location = useLocation();
+
+  useEffect(() => {
+    setLoggedIn(isLoggedIn());
+  }, [location]);
+  
```

Comment:
Any chance we could refactor this to use something like https://www.npmjs.com/package/react-cookie? It feels a bit odd to hook into location for this kind of thing.

Reply:
@KentShikama Can you tell me more about why this bothers you and what you would prefer to be done with this package?

Reply:
@KentShikama react-cookie wasn't going to work because it doesn't trigger a re-render when the cookies change, but I pushed another solution. Looks like I need to update so front end tests are passing though. Will re-request a review once I button that up.

Reply:
@trezp Pardon the delay in reply. It is just that the use of the `location` is "indirect". Ideally, we would be reacting to when the cookie is added/removed/expired as that is what the status of the login button should reflect.

Reply:
@trezp 

> react-cookie wasn't going to work because it doesn't trigger a re-render when the cookies change
Ah ok. Pardon for taking you down that path then.